keel-web/src/test/kotlin/com/netflix/spinnaker/keel/apidocs/ApiDocTests.kt

```diff
@@ -105,7 +112,7 @@ class ApiDocTests : JUnit5Minutests {
       .let { jacksonObjectMapper().readTree(it) }
   }
 
-  fun tests() = rootContext<Assertion.Builder<JsonNode>> {
+  fun tests() = SKIP - rootContext<Assertion.Builder<JsonNode>> {
```

Comment:
why did you need to skip this? I'd suggest putting a comment here..

Reply:
Because this is causing all PRs to fail and there's no one consuming the API docs. We had a discussion in Slack this morning, but the TL;DR is that it has to do with the ordering of `@Bean`s that customize the handling of types to produce the API docs, which is causing a `StackOverflowError`. In discussing with @robfletcher, we agreed to disable these tests temporarily until we can figure that out, since it's unrelated to changes in the PRs.

Reply:
Sure, can you add that as a comment please?